src/object-store/Cargo.toml

```diff
@@ -15,7 +15,7 @@ futures.workspace = true
 lazy_static.workspace = true
 md5 = "0.7"
 moka = { workspace = true, features = ["future"] }
-opendal = { version = "0.44", features = [
+opendal = { version = "0.44.1", features = [
```

Comment:
`0.44.1` is equivalent to `0.44` as per [cargo book](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html). If you want to upgrade opendal to at least `0.44.1`, you may resort to [tilde requirement](https://doc.rust-lang.org/cargo/reference/specifying-dependencies.html#tilde-requirements)

Reply:
Good point. I forget that point. It seems it's enough to upgrade only the lock file.